packages/frontend-v2/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: "next/core-web-vitals",
```

Comment:
is there a way to make it extend the workspace root eslint config as well?

Reply:
So the way I understand it, eslint cascades and merges eslint configs from the directory of the file all the way up to root. You can test this out by adding a unused var and you'll see the eslint rule "unused vars should be prefixed with an underscore" pop up and that rule is coming from the root eslint. I got this stuff from the [docs](https://eslint.org/docs/user-guide/configuring/configuration-files) under the `cascading and hierarchy` section but let me know if understood this wrong!